cdap-api/src/main/java/co/cask/cdap/api/customaction/AbstractCustomAction.java

```diff
@@ -15,6 +15,9 @@
  */
 package co.cask.cdap.api.customaction;
 
+import co.cask.cdap.api.annotation.TransactionControl;
+import co.cask.cdap.api.annotation.TransactionPolicy;
```

Comment:
It looks like these two classes are always being imported together. Can they be combined into one?